model_analyzer/config/generate/search_parameters.py

```diff
@@ -130,7 +130,7 @@ def _populate_parameters(self) -> None:
 
         if not self._is_composing_model:
             self._populate_concurrency()
```

Comment:
You need to check if request rate has been specified by the user:
There is a function in `config_command_profile::is_request_rate_specified()` that you should be using here

Reply:
yeah, I knew theoretically they shouldn't be set together, but I couldn't figure how to do the check. Thanks, I will make the changes.